actions/ci-beholder-validator/action.yml

```diff
@@ -0,0 +1,93 @@
+name: "Schema Validator Action"
+description: "Validates schema compatibility with schema registry"
+
+inputs:
+  docker-registry:
+    description: "Docker registry to use"
+    required: true
+    default: "aws"
+  image-tag:
+    description: "Image tag to use"
+    required: false
+    default: "latest"
+  # aws inputs
+  aws-role-duration-seconds:
+    description: ""
+    required: false
+    default: "900"
+  aws-region:
+    description: ""
+    required: false
+  aws-account-number:
+    description: ""
+    required: false
+  aws-role-arn:
+    description: ""
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout repo
+      uses: actions/checkout@v4.2.1
+      with:
+        fetch-depth: ${{ inputs.checkout-repo-fetch-depth }}
```

Comment:
1. Your README says to put a checkout before calling this action. If that's the case then this is unnecessary
2. This input doesn't exist

Reply:
I changed this. So this is not needed. Thank you will do this